app/util/session_id.py

```diff
@@ -0,0 +1,16 @@
+import uuid
+
+
+class SessionId(object):
+    SESSION_HEADER_KEY = 'Session-Id'
```

Comment:
Ah, okay. That spec above is just a draft so I'm not too concerned about overlap.